docs/first-flatpak.rst

```diff
@@ -0,0 +1,139 @@
+Building Your First Flatpak
+===========================
+
+This tutorial provides a quick introduction to Flatpak. In it, you will learn
+how to create a basic Flatpak application, which can be installed and run.
+
+To complete this tutorial, you should have installed Flatpak by following
+`the setup guide <https://flatpak.org/getting.html>`_. You will also need to
+haves the ``flatpak-builder`` tool, which is generally available as a package.
+
+1. Install a runtime and the matching SDK
+-----------------------------------------
+
+Flatpak requires every app to specify a runtime that it uses for its basic
+dependencies. Each runtime has a matching SDK (Software Development Kit), which
+contains all the things that are in the runtime, plus headers and development
+tools (similar to what is typically found in -devel/-dev packages in Linux
+distributions). This SDK is required to build apps for the runtime.
+
+In this tutorial we will use the Freedesktop runtime version 1.6. This runtime
+is provided by the Flathub repository. To add this, run:
+
+  $ flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+
+Then, to install the runtime and the SDK, run:
+
+  $ flatpak install flathub org.freedesktop.Platform//1.6 org.freedesktop.Sdk//1.6
+
+2. Create the app
+-----------------
+
+The app that is going to be created for this tutorial is a a simple script. To
+create it, copy the following to an empty file and save it as `hello.sh`:
+
+  #!/bin/sh
+  echo "Hello world, from a sandbox"
+
+3. Add a manifest
+-----------------
+
+Most Flatpaks are built using the `flatpak-builder` tool. This reads a manifest
+file which describes the key properties of the application and how it is to be
+built.
+
+To add a manifest to the hello world app, add the following to an empty file:
+
+  {
```

Comment:
This should be in a:

```
.. code-block:: json

```

Same with commands probably (sans json ofc)

Reply:
Sorry, don't know how I missed that. Fixed in the branch.